src/hooks/subscription-hooks.js

```diff
@@ -330,7 +333,8 @@ export function useTasksSubscription() {
     variables: subscriptionVariables,
   })
 
-  const tasks = data?.adjudicationRounds || null
+  const tasks =
+    data?.adjudicationRounds.map(transformRoundDataAttributes) || null
```

Comment:
See that we use the same round data transformations as we do for disputes.

Reply:
Ah gotcha, this wasn't being transformed before at all?

Reply:
No it wasn't.